dashboard/src/pages/ChapterDetails.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+<div v-if="chapter.doc" class="p-8 w-full">
+    <div v-if="showToast" class="z-10 absolute">
+        <Toast
+            :class="toastTitle == 'Success' ? 'bg-green-50 border-green-500' : 'bg-red-50 border-red-500'"
+            :icon="toastTitle == 'Success' ? 'check-circle' : 'x'"
+            icon-classes="stroke-2"
+            :title="toastTitle"
+            :text="toastMessage"
+            position="bottom-right"
+        ></Toast>
+    </div>
+    <div class="flex justify-between">
+        <div class="flex flex-col gap-3">
+            <FossClubBranding v-if="chapter.doc.chapter_type == 'FOSS Club'">{{ chapter.doc.chapter_type }}</FossClubBranding>
+            <CityCommunityBranding v-else>{{ chapter.doc.chapter_type.toUpperCase() }}</CityCommunityBranding>
```

Comment:
Typography transforms should be done with styles (text-transform)

Reply:
changed